econml/sklearn_extensions/linear_model.py

```diff
@@ -294,6 +305,17 @@ class WeightedMultiTaskLasso(WeightedModelMixin, MultiTaskLasso):
     n_iter_ : int | array-like, shape (n_targets,)
         number of iterations run by the coordinate descent solver to reach
         the specified tolerance.
+
+    .. testcode::
+        :hide:
+
+        # Our classes that derive from sklearn ones sometimes include
+        # inherited docstrings that have embedded doctests; we need the following imports
+        # so that they don't break.
+
+        import numpy as np
+        from sklearn.linear_model import lasso_path
+
```

Comment:
I think these should only need to be imported once.
```suggestion
```

Reply:
Unforutnately  now that each class is in a separate autosummary page, we need this to be on each page. We can no longer just put it at the top of the file for instance.